sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/PersonSteps.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.sormas.e2etests.steps.api;
+
+import cucumber.api.java8.En;
+import java.util.List;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.api.PersonsHelper;
+import org.sormas.e2etests.pojo.Person;
+import org.sormas.e2etests.state.ApiState;
+
+public class PersonSteps implements En {
+
+  @Inject
+  public PersonSteps(PersonsHelper personsHelper, ApiState apiState) {
+
+    When(
+        "API: I receive the person",
+        () -> {
+          List<String> personUuids = apiState.getResponse().jsonPath().get();
+          personsHelper.getPersonByUuid(personUuids.get(4717));
```

Comment:
What is this number are there so many persons in the list what happens if this list changes ?

Reply:
This was only for integration of API, it is not a real case.